### PR TITLE
Clarify Elasticsearch version support for 2.4.0

### LINKED ISSUE #7759: Feedback on page: /guides/v2.4/install-gde/system-requirements-tech.html
# General issue
Elasticsearch version requerment is incorect.
## Description:
Impossible to install Magento Comunity Edition with elasticsearch 7.6.x

## Possible solutions:
Minimum required version is 7.7.x

## Additional information:
`- magento/product-community-edition 2.4.0 requires elasticsearch/elasticsearch ~7.7.0 -> satisfiable by elasticsearch/elasticsearch[7.7.x-dev, v7.7.0] but these conflict with your requirements or minimum-stability.
- magento/product-community-edition 2.4.0 requires elasticsearch/elasticsearch ~7.7.0 -> satisfiable by elasticsearch/elasticsearch[7.7.x-dev, v7.7.0] but these conflict with your requirements or minimum-stability.
- Installation request for magento/product-community-edition 2.4.0 -> satisfiable by magento/product-community-edition[2.4.0].`

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) clarifies support for Elasticsearch 7.6.x in the 2.4.0 system requirements. We need to emphasize the fact that Magento is tested on ES 7.6.x.

It also adds a note at the top of the page redirecting cloud users to the Service versions section in the cloud guide since there can sometimes be differences between cloud and on-prem.

Fixes #7759 

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html#elasticsearch

### CODE REVIEW

**Comment by hguthrie** [comment]
I don't know if it is important to note this here. If they are using Cloud, there may be some problems with 7.6.x.
See Cloud team dev pull request #7752

**Comment by meker12** [comment]
> I don't know if it is important to note this here. If they are using Cloud, there may be some problems with 7.6.x.
> See Cloud team dev pull request #7752

I checked in with @NadiyaS . That pull request is on hold, and I'm not sure what the final outcome of the discussion is. Cloud version support creates confusion because they support different things than Core. For example, they maintain support for EOL versions for backward-compatibility with existing projects. Not sure how to handle this in Core docs-- should we have some note that says for info on support on Cloud, see Cloud Guide?

**Comment by jeff-matthews** [comment]
Thanks @hguthrie & @meker12. I think it probably is a good idea to add a note about cloud requirements and linking to the version support matrix in the cloud guide.

File: src/guides/v2.4/install-gde/system-requirements-tech.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ functional_areas:
   - Setup
 ---
 
+{:.bs-calloout-info}
+If you are working on a {{site.data.var.ece}} project, see [Service versions]({{ site.baseurl }}/cloud/project/project-conf-files_services.html#service-versions).
```
Comment (meker12) [inline src/guides/v2.4/install-gde/system-requirements-tech.md:11]:
```suggestion
If you are working on a {{site.data.var.ece}} project, see [Service versions]({{ site.baseurl }}/cloud/project/project-conf-files_services.html#service-versions) in the _Cloud Guide_.
```

**Comment by jeff-matthews** [comment]
Thanks @meker12!

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**